modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -321,6 +321,11 @@ export interface RecoverTokenTransaction {
   };
 }
 
+export interface TokenEnablementConfig {
+  requiresTokenEnablement: boolean;
+  supportsMultipleTokenEnablements?: boolean;
```

Comment:
add doc: "flag indicating if multiple tokens can be enabled in a single transaction"

Reply:
also, this should be required and we default to false.